File: python/prime_number_algorithm.py
# create an algorithm that finds prime numbers

def is_prime(number):
  if number == 0 or number == 1:
      print(number, 'is not prime') 
      return

  for i in range(2, number-1, 1):
    if (number % i) == 0:
      print(number, 'is not prime')
      return
  print(number, 'is prime')

is_prime(0)
is_prime(1)
is_prime(23)
is_prime(31)
is_prime(50)
is_prime(53)
is_prime(76)
is_prime(99)
is_prime(1000)
is_prime(1777)
is_prime(2500)
is_prime(5743)
is_prime(22225)
is_prime(28663)

File: javascript/prime_number_algorithm.js
// create an algorithm that finds prime numbers

function isPrime(number) {
    if (number == 0 || number == 1) {
        return `${number} is not prime`;
    }
    
    for (var i = 2; i < number; i++) {
        if ((number % i) == 0) {
            return `${number} is not prime`;
        }
    }

    return `${number} is prime`;
}

console.log(isPrime(0));
console.log(isPrime(1));
console.log(isPrime(23));
console.log(isPrime(31));
console.log(isPrime(50));
console.log(isPrime(53));
console.log(isPrime(76));
console.log(isPrime(99));
console.log(isPrime(1000));
console.log(isPrime(1777));
console.log(isPrime(2500));
console.log(isPrime(5743));
console.log(isPrime(22225));
console.log(isPrime(28663));
